template-parts/content-alternate.php

```diff
@@ -3,18 +3,50 @@
  * Template part for displaying alternate posts
  *
  * @package siteorigin-unwind
- * @since siteorigin-unwind 1.1 
+ * @since siteorigin-unwind 1.1
  * @license GPL 2.0
  */
+
+$gallery = get_post_gallery( get_the_ID(), false );
```

Comment:
To keep things a little more neat, this whole block should be handled in a separate function in the inc folder. This is especially important because we have the exact same logic in content-grid.php.